libs/gltfio/include/gltfio/FilamentAsset.h

```diff
@@ -334,6 +334,23 @@ class UTILS_PUBLIC FilamentAsset {
     void addEntitiesToScene(filament::Scene& targetScene, const Entity* entities, size_t count,
             SceneMask sceneFilter);
 
+    /**
+     * Releases ownership of entities and their Filament components.
+     *
+     * This makes the client take responsibility for destroying Filament
+     * components (e.g. Renderable, TransformManager component) as well as
+     * the underlying entities.
+     */
+    void detachFilamentComponents();
```

Comment:
Is the name "Filament" required here?

Reply:
Imp uses our ECS, this makes it clear that Imp components are not being detached.

Reply:
👍